test/e2e/marketplace_test.go

```diff
@@ -50,6 +46,7 @@ func TestMarketplace(t *testing.T) {
 	if err != nil {
 		t.Fatalf("failed to add catalogsourceconfig custom resource scheme to framework: %v", err)
 	}
+
```

Comment:
Why? Please don't add and remove empty lines because you prefer it that way. We can come up with a guideline for such things and fix all of this in a separate PR.

Reply:
Addressed in the latest commit.